src/js/components/Grommet/GrommetContext.js

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+// Track which components mounted onto the DOM are coming from
+// Grommet (see FocusedContainer.js)
+export const GrommetContext = React.createContext(undefined);
```

Comment:
Wondering if "GrommetContext" is the right term here, perhaps "RootsContext"?

Reply:
Good comment, I was trying to think what would be appropriate an appropriate name. I could see `Roots` working but it didn't click right off the bat. Let me brainstorm a little more while I address the other comments.

Reply:
I've gone with RootsContext. I think once I equated it to the concept of "root directory", it makes more sense. The other idea I had was `OriginContext`, but that didn't have any programming equivalent so it doesn't feel like it's a better option.